renderer/src/components/ActivityChartCard.jsx

```
@@ -106,119 +108,85 @@ function ActivityChartCard({ title, description, chartType }) {
+            No chart available
+          </div>
+        )}
+      </div>
+
+      {/* Debug Information */}
+      {debugInfo && (
+        <div className="mt-4">
+          <button
+            className="flex items-center text-sm text-gray-600 hover:text-gray-800"
+            onClick={() => setDebugInfo(null)}
           >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)